automated install/basic-install.sh

```diff
@@ -2704,7 +2867,7 @@ main() {
 
         # Get the privacy level if it exists (default is 0)
         if [[ -f "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" ]]; then
-            PRIVACY_LEVEL=$(sed -ne 's/PRIVACYLEVEL=\(.*\)/\1/p' "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf")
+            PRIVACY_LEVEL=$(sed -n -e 's/PRIVACYLEVEL=\(.*\)/\1/p' "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf")
```

Comment:
Please use the full flags instead of the shorthands if you'd like to split them up.

Reply:
I once tried a `sed` command and it didn't work just because some options were merged
Something like `sed -iEe '...' ...`